regolith/schemas.py

```diff
@@ -2865,32 +2902,49 @@
             "description": "indicating the title of the presentation along with the link and the presenter ",
             "required": False,
             "type": "dict",
+            "schema": {
+                 "title": {
+                  "description": "The title of the presentation.  tbd if it is not known yet",
+                  "type": "string",
+                  "required": True
+                 },
+                 "link": {
+                  "description": "The name of the presentation repo in gitlab. e.g., 2003sb_grpmtg.  It is assumed that it will be in the talks group on gitlab.  tbd if it is not known yet",
```

Comment:
Please change the description.  This has now changed it is "the url to the repo, google slide location, or other web location where the presentation can be found"

Reply:
Done